edxval/migrations/0008_auto_20170922_0238.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
```

Comment:
This would also need to change as 0006 migration has been renamed in https://github.com/edx/edx-val/pull/102